runner/components/io.siddhi.distribution.metrics.core/src/test/java/io/siddhi/distribution/metrics/core/StatisticsTestCase.java

```diff
@@ -47,6 +46,7 @@
  * Test case for carbon metrics inside siddhi.
  */
 public class StatisticsTestCase {
+
```

Comment:
Remove all unnecessary changes

Reply:
The above comment is addressed by the commit 47afa97.